metadata_backend/api/handlers/object.py

```diff
@@ -283,3 +305,43 @@ async def patch_object(self, req: Request) -> Response:
         body = ujson.dumps({"accessionId": accession_id}, escape_forward_slashes=False)
         LOG.info(f"PATCH object with accession ID {accession_id} in schema {collection} was successful.")
         return web.Response(body=body, status=200, content_type="application/json")
+
+    async def prepare_folder_patch(self, schema: str, ids: List, params: Dict[str, str]) -> List:
+        """Prepare patch operations list.
+
+        :param schema: schema of objects to be added to the folder
+        :param ids: object IDs
+        :param params: addidtional data required for db entry
+        :returns: list of patch operations
+        """
+        if not params.get("cont_type", None):
+            submission_type = "Form"
+        else:
+            submission_type = params["cont_type"].upper()
```

Comment:
realised that for this to work we need to add `CSV` to lines:
- https://github.com/CSCfi/metadata-submitter/blob/develop/metadata_backend/helpers/schemas/folders.json#L999
- https://github.com/CSCfi/metadata-submitter/blob/develop/metadata_backend/helpers/schemas/folders.json#L1037

Reply:
strangely it worked without schema changes.